examples/readers/src/llamaparse-docx.ts

```diff
@@ -1,4 +1,4 @@
-import { Language, LlamaParseReader } from "llamaindex";
+import { Language, LlamaParseReader } from "@llamaindex/cloud/reader";
```

Comment:
i think that's an error - can we move that reader to the cloud package
```suggestion
import { Language, LlamaParseReader } from "@llamaindex/cloud";
```

Reply:
remember: idea is that we package everything from one vendor into one package

Reply:
this PR should not be breaking, so just reexport as `import { Language, LlamaParseReader } from "@llamaindex/cloud"` here